easybuild/easyconfigs/s/SuperLU_DIST/SuperLU_DIST-5.3.0-intel-2020a.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = "EB_SuperLU"
+
+name = 'SuperLU_DIST'
+version = '5.3.0'
```

Comment:
There's a more recent 5.3.0 variant available tagged as `5.3.0-trisolve`, maybe using that one is better?

See https://github.com/xiaoyeli/superlu_dist/compare/v5.3.0...v5.3.0-trisolve

BTW, I think it makes sense to throw this easyconfig (and the change in the test suite) in a separate PR...

Reply:
Ye, it makes sense, will create another PR.